lib/smart_proxy_remote_execution_ssh/api.rb

```diff
@@ -71,6 +71,14 @@ class Api < ::Sinatra::Base
         end
       end
 
+      get "/jobs/:job_uuid/cancel" do |job_uuid|
+        do_authorize_with_ssl_client
+
+        with_authorized_job(job_uuid) do |job_record|
+          {}
```

Comment:
Why do we need an empty endpoint?

Reply:
This just just a workaround to yggdrasil doing things its own way. Upon receiving a message, yggdrasil *needs* to call an url, otherwise it just fails :/